Source/Core/DolphinQt2/WiiUpdate.cpp

```diff
@@ -74,46 +119,25 @@ void PerformOnlineUpdate(const std::string& region, QWidget* parent)
               true);
           return !was_cancelled.IsSet();
         },
-        region);
+        std::forward<Args>(args)...);
     Core::QueueHostJob([&dialog] { dialog.close(); }, true);
     return res;
   });
 
   dialog.exec();
+  Core::HostDispatchJobs();
```

Comment:
Does this need to be explicit here? DolphinQt2 will run this function whenever it has spare time in the event loop.